aiohttp/locks.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+import collections
+
+from .helpers import create_future
+
+
+class Event:
```

Comment:
Isn't it possible to inherint from asyncio.Event and override set method instead? I'm not sure we want to maintain whole own Even lock.